components/Text.js

```diff
@@ -23,6 +23,7 @@ export const P = styled.p.attrs(props => ({
       white-space: nowrap;
       overflow: hidden;
     `}
+  ${props => props.wordBreak}
```

Comment:
`wordBreak` prop is already supported by this component, see https://github.com/opencollective/opencollective-frontend/pull/5395/files#diff-483920b66c212791608965bc62aa404cd8b73051c25192b68a856cd6ecb4e7b4R17

Reply:
Alright. I've taken note of it.